cypress/e2e/interface_builder.cy.js

```diff
@@ -597,6 +603,7 @@ describe('Interface builder tests', () => {
         interfaceFixtures.forEach((interfaceFixture) => {
           cy.fixture(interfaceFixture).then(({ data: iface }) => {
             setupInterfaceEditorFromSource(iface);
+            setupInterfaceEditorFromUI(iface);
```

Comment:
This looks like cheating :)

This test `correctly loads interface from its source` should validate that when the user writes into the editor (`setupInterfaceEditorFromSource(iface)`) then the UI form fields are automatically updated and correspond to the provided source.

If we call `setupInterfaceEditorFromUI(iface)` to setup the UI form fields, of course they will correspond: it defies the purpose of the test.

If we want to maintain the same behavior, we can update the implementation instead of the test, making sure that when the user changes the editor's content then the UI fields are updated.